web/src/components/Modal.tsx

```diff
@@ -62,9 +62,9 @@ const Modal = ({ title, closeModal, children, direction, wrapInPortal = false }:
 
     return () => layoutElement?.setAttribute('aria-hidden', 'false')
   }, [])
-
+  // display check option is needed for portals - https://github.com/focus-trap/tabbable/blob/master/CHANGELOG.md#532
```

Comment:
I think you linked the wrong release, shouldnt it be https://github.com/focus-trap/tabbable/blob/master/CHANGELOG.md#600?

It says
> restore old (incorrect) behavior

Should we investigate a better fix?

Reply:
Gonna adjust the link. Yes I can create an New issue for that. I think we should use this fix as a workaround